src/odemis/acq/fastem.py

```diff
@@ -349,16 +372,16 @@ def run(self):
         # Update the position of the first tile.
         self._pos_first_tile = self.get_pos_first_tile()
 
-        if self._pre_calibrate:
+        if self._pre_calibrations:
             # The pre-calibrations should run on a position that lies a full field
             # outside the ROA, therefore temporarily set the overlap to zero.
             overlap_init = self._roa.overlap
             self._roa.overlap = 0
             # Move the stage to the tile with index (-1, -1), to ensure the autofocus and image translation pre-align
             # are done to the top left of the first field, outside the region of acquisition to limit beam damage.
             self.field_idx = (-1, -1)
-            self.move_stage_to_next_tile()
-            self.pre_calibrate()
+            stage_pos_x, stage_pos_y = self.get_abs_stage_movement()
+            self.pre_calibrate(self._pre_calibrations, (stage_pos_x, stage_pos_y))
```

Comment:
If you do not use my previous comment you can at least update these two lines. I think this should also work:

```
stage_pos = self.get_abs_stage_movement()
self.pre_calibrate(self._pre_calibrations, stage_pos)
```